### DISCUSSION TITLE
code in colab

### DISCUSSION THREAD

**Discussion by aipdv** [python-telegram-bot/python-telegram-bot#4169] [Q&A]
i'm just making my own telegram chat bot but when i tried to run it in google colab notebook, its giving me multiple errors... the problem arises when im declaring multiple async functions for different commandHandlers... Anyone has any trick to run it in colab>>>?

**Comment by Bibo-Joshi** [comment]
Please have a look at https://github.com/python-telegram-bot/python-telegram-bot/issues/3251
